playground/node-compat/__tests__/worker-random/random.spec.ts

```diff
@@ -3,15 +3,18 @@ import { getJsonResponse } from '../../../__test-utils__';
 
 // Disabling actually querying the database in CI since we are getting this error:
 // > too many connections for role 'reader'
-test.runIf(!process.env.CI)(
-	'should be able to call `getRandomValues()` bound to any object',
-	async () => {
-		const result = await getJsonResponse();
-		expect(result).toEqual([
-			expect.any(String),
-			expect.any(String),
-			expect.any(String),
-			expect.any(String),
-		]);
-	},
-);
+// TODO: reintroduce test
+test
+	.runIf(!process.env.CI)
```

Comment:
Actually (not related to this PR) this condition is not relevant to this test. It was copied incorrectly from the postgres test. We should remove this and also the comment above.